qemu/tests/block_copy.py

```diff
@@ -136,6 +136,40 @@ def is_cancelled():
         if self.vm.monitor.protocol == "qmp":
             self.vm.monitor.clear_event("BLOCK_JOB_CANCELLED")
 
+    def is_paused(self):
+        """
+        Return block job paused status.
+        """
+        paused = self.get_status().get("paused")
+        offset_p = self.get_status().get("offset")
+        time.sleep(random.uniform(1, 3))
+        offset_l = self.get_status().get("offset")
+        paused &= bool(offset_p == offset_l)
+        return paused
+
+    def pause_job(self):
+        """
+        pause active job;
+        """
+        if self.is_paused():
```

Comment:
@xutian To pause a paused job wont cause any error for qemu, so I think there is no any bug for this funciton for qemu. Keep this check just for informing tester job already paused.